docs/user-guide/data-structures/data-structures.ipynb

```diff
@@ -106,72 +108,24 @@
     "var.values"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "print(var.variances)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Variances must have the same shape as values, and units are specified using the [scipp.units](../reference/units.rst) module or with a string:"
```

Comment:
I think we should keep discussing units on this page. It is such a core concept of scipp. Maybe add one to the first example of creating a variable?